optaplanner-examples/src/main/java/org/optaplanner/examples/investment/optional/score/InvestmentConstraintProvider.java

```diff
@@ -0,0 +1,61 @@
+package org.optaplanner.examples.investment.optional.score;
+
+import java.util.function.Function;
+
+import org.optaplanner.core.api.score.buildin.hardsoftlong.HardSoftLongScore;
+import org.optaplanner.core.api.score.stream.Constraint;
+import org.optaplanner.core.api.score.stream.ConstraintCollectors;
+import org.optaplanner.core.api.score.stream.ConstraintFactory;
+import org.optaplanner.core.api.score.stream.ConstraintProvider;
+import org.optaplanner.core.api.score.stream.Joiners;
+import org.optaplanner.examples.investment.domain.AssetClassAllocation;
+import org.optaplanner.examples.investment.domain.InvestmentParametrization;
+import org.optaplanner.examples.investment.domain.Region;
+import org.optaplanner.examples.investment.domain.Sector;
+
+public class InvestmentConstraintProvider implements ConstraintProvider {
+
+    private static final String CONSTRAINT_PACKAGE = "org.optaplanner.examples.investment.solver";
+
+    @Override
+    public Constraint[] defineConstraints(ConstraintFactory constraintFactory) {
+        // TODO Auto-generated method stub
+        return new Constraint[]{
+                                assetsDeviationGreaterThanMaximumPenalty(constraintFactory),
+                                regionQuantityGreaterThanMaximumPenalty(constraintFactory),
+                                sectorQuantityGreaterThanMaximumPenalty(constraintFactory),
+                                assetExpectedReturnReward(constraintFactory)
```

Comment:
Wrong indentation.
Also, I'm pretty sure some of the lines in this file exceed the 120-char limit.

Please configure your IDE according to the accepted code style.

Reply:
Again, this is KIE Java Conventions for Eclipse; will write a Dockerfile to run the IDEA formatter on a directory.

Reply:
Done, configured a git hook to auto run IDEA formatter on changed files before commit in all KIE repo's.